web/src/compose_reply.js

```diff
@@ -42,6 +43,10 @@ export function respond_to_message(opts) {
         }
         message = message_opts.message;
     } else {
+        if (message_lists.current === undefined) {
+            blueslip.error("message_lists.current expected to be defined for current hash." + window.location.hash);
+        }
```

Comment:
I think this might be safe enough to do an assert -- is there a reason we shouldn't do that?